src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -139,6 +139,7 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
                 table_format=describe.get("Provider", "").upper(),
                 location=describe.get("Location", None),
                 view_text=describe.get("View Text", None),
+                table_properties=describe.get("Table Properties", None),
```

Comment:
This code path is only triggered for testing, production table uses tables.scala, remove this line